src/swagger-specs/flow-service.yaml

```diff
@@ -1311,172 +1256,378 @@ definitions:
+                            example: "false"
+                            description: Defines if the data can be previewed.
+                      canFetchSchema:
+                        type: object
+                        properties:
+                          canFetchSchema:
+                            type: boolean
+                            example: "false"
+                            description: Defines if the schema of the fields of the data can be fetched.
   PermissionDTO:
     type: object
+    description: Contains information regarding the permissions applied to the queried source.
     properties:
-      '@type':
-        type: string
-        example: lowlevel
-      name:
-        type: string
-        example: EnterpriseSource
-      permissions:
+      manage:
         type: array
+        description: Defines the type of access available to the `manage` permission.
         items:
-          type: string
-          enum:
-            - read
-            - write
+          type: object
+          properties:
+            '@type':
+              type: string
+              example: lowlevel
+              description: Defines the permission level required for the source.
+            name:
+              type: string
+              example: EnterpriseSource
+              description: Defines the integration type of the source.
+            permissions:
+              type: array
+              description: Defines the actions a user can do, given the `manage` permission.
+              items:
+                type: string
+                enum:
+                  - write
+      view:
+        type: array
+        items:
+          type: object
+          description: Defines the type of access available to the `view` permission.
+          properties:
+            '@type':
+              type: string
+              example: lowlevel
+              description: Defines the permission level required for the source.
+            name:
+              type: string
+              example: EnterpriseSource
+              description: Defines the integration type of the source.
+            permissions:
+              type: array
+              description: Defines the actions a user can do, given the `manage` permission.
+              items:
+                type: string
+                enum:
+                  - read
   SourceSpecDTO:
     type: object
     properties:
-      schema:
+      name: 
         type: string
-        example: "http://json-schema.org/draft-07/schema#"
+        example: "CloudStorage"
+        description: The name of the type of the source.
       type:
         type: string
-        example: "object"
-      description:
-        type: string
-        example: "Defines params required when this connection is used as a source in flow"
-      path:
+        example: "CloudStorage"
+        description: The type of the source. This value is the same as the category assigned to the source.
```

Comment:
"The type of source." I'm not 100% sure if "the" should precede "source" or not.